storybooknative/src/InputField/index.js.flow

```diff
@@ -15,10 +14,10 @@ export type Props = {|
   +inlineLabel?: boolean,
   +value?: string,
   +placeholder?: string,
```

Comment:
placeholder also needs to be a component, see 
https://github.com/kiwicom/mobile/blob/master/packages/shared/src/forms/TextInput.js

Reply:
I will do a separate PR for placeholder.